template/src/.vitepress/theme/style.css

```diff
@@ -1,122 +1,106 @@
+/* Customize default theme styling by overriding CSS variables:
+https://github.com/vuejs/vitepress/blob/main/src/client/theme-default/styles/vars.css */
+/* Example */
+/* https://github.com/vuejs/vitepress/blob/main/template/.vitepress/theme/style.css */
+
 .VPHero .clip {
   white-space: pre;
   max-width: 600px;
 }
 
 /* Fonts */
-
 @font-face {
-    font-family: JuliaMono-Regular;
-    src: url("https://cdn.jsdelivr.net/gh/cormullion/juliamono/webfonts/JuliaMono-Regular.woff2");
+  font-family: JuliaMono-Regular;
+  src: url("https://cdn.jsdelivr.net/gh/cormullion/juliamono/webfonts/JuliaMono-Regular.woff2");
 }
 
 :root {
-  /* Typography */
-  --vp-font-family-base: "Barlow", "Inter var experimental", "Inter var",
-    -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu,
-    Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+/* Typography */
+--vp-font-family-base: "Barlow", "Inter var experimental", "Inter var",
+  -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu,
+  Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
 
-  /* Code Snippet font */
-  --vp-font-family-mono: JuliaMono-Regular, monospace;
+/* Code Snippet font */
+--vp-font-family-mono: JuliaMono-Regular, monospace;
 }
 
 /* Disable contextual alternates (kind of like ligatures but different) in monospace, 
-   which turns `/>` to an up arrow and `|>` (the Julia pipe symbol) to an up arrow as well. */
+ which turns `/>` to an up arrow and `|>` (the Julia pipe symbol) to an up arrow as well. */
 .mono-no-substitutions {
-  font-family: "JuliaMono-Regular", monospace;
-  font-feature-settings: "calt" off;
+font-family: "JuliaMono-Regular", monospace;
+font-feature-settings: "calt" off;
 }
 
 .mono-no-substitutions-alt {
-  font-family: "JuliaMono-Regular", monospace;
-  font-variant-ligatures: none;
+font-family: "JuliaMono-Regular", monospace;
+font-variant-ligatures: none;
 }
 
 pre, code {
-  font-family: "JuliaMono-Regular", monospace;
-  font-feature-settings: "calt" off;
+font-family: "JuliaMono-Regular", monospace;
+font-feature-settings: "calt" off;
 }
 
 /* Colors */
-
 :root {
   --julia-blue: #4063D8;
   --julia-purple: #9558B2;
   --julia-red: #CB3C33;
   --julia-green: #389826;
 
-  --vp-c-brand: #40a52f;
-  --vp-c-brand-light: #3dd027;
-  --vp-c-brand-lighter: #9499ff;
-  --vp-c-brand-lightest: #bcc0ff;
-  --vp-c-brand-dark: #535bf2;
-  --vp-c-brand-darker: #454ce1;
+  --vp-c-brand: #0087d7;
```

Comment:
is this changing the default colors?

Reply:
a little bit, we will see just a different transition in get started button in the light-mode. Which, in general is off in the current version. This PR makes dark and light modes consistent 😄 . Also, I tried to remove as much as possible of the css unused.